# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

bool trajectory_end_reached
int8 number_of_remaining_trajectory_points

